src/containers/app/AppHeader.js

```diff
@@ -45,7 +46,18 @@ const Menu = styled.div`
 `;
 
 const MenuInternalLink = styled(NavLink)`
-  color: ${NEUTRALS.WHITE};
+  color: ${NEUTRALS.GRAY01};
+  margin-right: 30px;
+  text-decoration: none;
+
+  &:hover {
+    color: ${NEUTRALS.GRAY01};
+    cursor: pointer;
+  }
+`;
+
+const MenuInternalHashLink = styled(NavHashLink)`
```

Comment:
Aha, now these styles look a lottt like the `MenuInternalLink`, right? We can do a fancy thing that will apply styles to each element without repeating the code:
```
const getLinkStyles = () => (
   css`
   color: ${NEUTRALS.GRAY01};
   margin-right: 30px;
   text-decoration: none;

   &:hover {
     color: ${NEUTRALS.GRAY01};
     cursor: pointer;
   }
  `
);

const LinktoHome = styled(NavLink)`
  ${getLinkStyles}
`;

const MenuInternalHashLink = styled(NavHashLink)`
  ${getLinkStyles}
`;

Reply:
You just also need to import `css` from `styled-components`!

```
import styled, { css } from 'styled-components';

Reply:
wow so good!